ajax/libs/nes.css/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "nes.css",
+  "filename": "nes.min.css",
+  "description": "NES.css is NES-style CSS Framework.",
+  "author": "BcRikko (https://github.com/Bcrikko)",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nostalgic-css/NES.css"
+  },
+  "license": "MIT",
+  "keywords": [
+    "css",
+    "scss",
+    "nes",
+    "framework"
+  ],
+  "npmName": "nes.css",
+  "npmFileMap": [
+    {
+      "basePath": "css",
+      "files": [
+        "*/"
```

Comment:
Let's use `nes*` to prevent including something we don't want.

Reply:
Thanks for the review. Updated as per review comments.